### PR TITLE
Adds documentation about stop watching for data changes

### PR DESCRIPTION
**Pull request by albertmourato**

### CODE REVIEW

File: node/electric/src/pages/docs/data/real-time-data/index.md [hunk 1]
```diff
@@ -312,6 +312,33 @@ For more examples of useful filters and result sorting you can use, see our **[F
...
+    .on('update', function(data) {
+      console.log("Data updated: " + data);
+    });
+```
+
+Once you have the reference of the watcher, all you have to do is call the `off` method and choose which kind of modification you want to stop watching for.
+
+```javascript
+watcher
+  .off('update')
```
Comment (zenorocha) [inline node/electric/src/pages/docs/data/real-time-data/index.md]: Can you put on the same line?

File: node/electric/src/pages/docs/data/real-time-data/index.md [hunk 2]
```diff
@@ -312,6 +312,32 @@ For more examples of useful filters and result sorting you can use, see our **[F
 
 </article>
 
+<article id="7">
+
+## Stop Watching for Data Changes
+
+In some cases, you might need to stop the data watcher. To do that, you should save a reference to the watcher in a variable as shown bellow.
```
Comment (jonnilundy) [inline node/electric/src/pages/docs/data/real-time-data/index.md]: bellow > below

File: node/electric/src/pages/docs/data/real-time-data/index.md [hunk 3]
```diff
@@ -312,6 +312,32 @@ For more examples of useful filters and result sorting you can use, see our **[F
...
+var watcher = 
+  WeDeploy
+    .data('https://<serviceID>-<projectID>.wedeploy.io')
+    .watch('movies')
+    .on('update', function(data) {
+      console.log("Data updated: " + data);
+    });
+```
+
+Once you have the reference of the watcher, all you have to do is call the `off` method and choose which kind of modification you want to stop watching for.
```
Comment (jonnilundy) [inline node/electric/src/pages/docs/data/real-time-data/index.md]: of the watcher > to the watcher
modification > change

**Comment by albertmourato** [comment]
@jonnilundy changes applied

**Pull request merged**